README.md

```diff
@@ -37,6 +37,9 @@ with an `ini` file for command line options:
 broker = "//kafkabroker:9092"
 status-topic = "//kafkabroker:9092/the_status_topic"
 streams-json = ./streams.json
+kafka-config=consumer.timeout.ms 501 fetch.message.max.bytes 1234 api.version.request true
```

Comment:
Hm, isn't it way harder to read this INI style format compared to the former JSON, or is it only me?

Isn't it also harder for the machine? JSON maps easily in and out of Python, but this kind of configuration format needs explicit conversion logic.

Reply:
Yes it is only you. JSON is scourge upon humanity when used for application configuration (see [this](https://arp242.net/weblog/json_as_configuration_files-_please_dont), [this](https://www.lucidchart.com/techblog/2018/07/16/why-json-isnt-a-good-configuration-language/) and [this](https://revelry.co/json-configuration-file-format/) link), the ini files are almost always easier to read and the format allows comments!